src/engine/tests/Test_CertificateNode.h

```diff
@@ -0,0 +1,200 @@
+/*********************                                                        */
+/*! \file BoundExplainer.h
+ ** \verbatim
+ ** This file is part of the Marabou project.
+ ** Copyright (c) 2017-2019 by the authors listed in the file AUTHORS
+ ** in the top-level source directory) and their institutional affiliations.
+ ** All rights reserved. See the file COPYING in the top-level source
+ ** directory for licensing information.\endverbatim
+ **
+ ** [[ Add lengthier description here ]]
+ **/
+
```

Comment:
The test file name must match the .h/cpp file names. There's no file called CertificateNode.